edx/xblock-lti-consumer
# [BD-24][TNL-7977][BB-3789]: General improvements to LTI 1.3 implementation

This PR improves logging and adds more information to the studio UI:
- d49965b925753c3fc2dacb86b6d4343b2b7e805d: Improve error and exception logging and pass tracebacks forward. Use the django built in functions for 404 errors instread of `HttpResponse`
- 3492fa2dd5d1ad3bd90d5ca02236bacee08b4a99: Remove misleading LTI information from Studio UI and more information when Deep linking is already set up.
- 65d8096bfbdbc3ab915d1bff083914a2ee171764: Internationalize LTI 1.3 templates
- 065de1c778b85f5ad74ecfb5b4c99f829863c9a5: Add i18n tooling

**Testing instructions:**
1. Checkout this branch.
2. Set up an LTI tool that **supports deep linking** (either IMS tool with user logged in or django pylti tool).
3. Check the studio view.
4. Run a deep linking launch.
5. Refresh the studio view page and check that the LTI Deep Linking information is displayed. 

**Reviewers:**

**Useful links:**
- https://openedx.atlassian.net/wiki/spaces/COMM/pages/753532962/Native+XBlocks+Morning+Session
- https://github.com/edx/i18n-tools
